README.md

```diff
@@ -8,28 +8,36 @@ Govdirectory aims to become a global directory of government agencies and their
 Wikidatas community, sourceability, versioning, and potential custom tooling that we create will ensure that not only should the information be correct but when it isn't, because of vandalism or something else, users will be able, and suggested, to validate the information.
 The goal is for this directory to be useful to journalists, web-archivists, researchers and many others, certainly including regular citizens.
 
-## Install
+## Development setup
 
-### Prerequisites
+Govdirectory is a static site meaning that it already has all of its pages generated when a visitor visits it. Govdirectory uses Snowman and SPARQL to do this.
 
- - [Snowman](https://github.com/glaciers-in-archives/snowman)
- - A local Wikidata Query Service (WDQS) instance or a Snowman cache directory.
+Because each initial build of Snowman issues thousands (yes thousands) of SPARQL queries one should never an **initial** build against `query.wikidata.org` but rather against a local WDQS instance. However, because setting up a WDQS instance is nontrival we provide a copy of a [Snowman build cache directory](https://github.com/govdirectory/website-cache), if in use, this will ensure Snowman only queries Wikidata when a query is updated.
```

Comment:
```suggestion
Because each initial build of Snowman issues thousands (yes thousands) of SPARQL queries one should never make an **initial** build against `query.wikidata.org` but rather against a local WDQS instance. However, because setting up a WDQS instance is nontrival we provide a copy of a [Snowman build cache directory](https://github.com/govdirectory/website-cache). If in use, this will ensure Snowman only queries Wikidata when a query is updated.
```